### DISCUSSION TITLE
Eigene Fahrten eintragen

### DISCUSSION THREAD

**Discussion by Erikp02** [Traewelling/traewelling#2204] [Ideas / Feature Requests]
Hallo,

eine Idee für eine Funktion wäre, dass man seine eigenen Fahrten in Träwelling eintragen kann, also wenn man zum Beispiel mit dem Fernbus fährt, da diese nicht auswählbar sind, dass man einfach den Startort (Dresden Hauptbahnhof) und den Zielort (Berlin ZOB) und die jeweilige Uhrzeit einträgt.

Ich nenne mal ein Beispiel, wie das aussehen kann

**Art der Gattung** (ICE, FLX, RE usw.)
**Fahrtnummer**
**Einstiegsort**
**Abfahrtszeit** (sollte auch anpassbar sein, wegen evtl. Verspätungen)
**Ausstiegsort**
**Ankunftszeit** (siehe Abfahrtszeit)

Ich hoffe, ich habe es soweit verständlich erklärt.

Mit freundlichen Grüßen
Erik

**Comment by MrKrisKrisu** [comment]
Hi! So ein Feature wird kommen. Wir bereiten aktuell schon im Hintergrund etwas unser Backend darauf vor und testen die Features bereits. Es wird demnächst auch eine Schnittstelle für Drittanwendungen zum testen geben. Das wird sich aber zunächst nur auf den Start- und Zielort an Träwelling bereits bekannten Bus-/Bahn/...-Stationen beschränken.
